app/models/order.rb

```diff
@@ -14,4 +14,17 @@ class Order < ApplicationRecord
   validates :delivery_time_zone, presence: true
   validates :delivery_address, presence: true, length: { maximum: 200 }
   validates :addressee_name, presence: true, length: { maximum: 50 }
+  validate :available_date_of_delivery
+
+  def self.available_dates_of_delivery
+    available_start_date_of_delivery = 3.business_days.from_now.to_date
+    available_end_date_of_delivery = 15.business_days.from_now.to_date
+    available_start_date_of_delivery.business_dates_until(available_end_date_of_delivery)
+  end
+
+  def available_date_of_delivery
+    unless Order.available_dates_of_delivery.include?(delivery_on)
```

Comment:
unlessのほうが、否定条件に読めるのでエラーケースではベターかもですね。

ちなみに、if-exclude? でも書けるので、ケースバイケースで使い分けてみて。
```suggestion
    if Order.available_dates_of_delivery.exclude?(delivery_on)
```

Reply:
`Date::between?` をつかうとスッキリしそう。
```rb
unless delivery_on.between?(3.business_day.from_now.to_date, 15.business_day.from_now.to_date)
```